contracts/pods/PodFactory.sol

```diff
@@ -0,0 +1,242 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import {IGnosisSafe} from "./orcaInterfaces/IGnosisSafe.sol";
+import {IControllerV1} from "./orcaInterfaces/IControllerV1.sol";
+import {IMemberToken} from "./orcaInterfaces/IMemberToken.sol";
+import {IPodFactory} from "./IPodFactory.sol";
+
+import {TribeRoles} from "../core/TribeRoles.sol";
+import {OptimisticTimelock} from "../dao/timelock/OptimisticTimelock.sol";
+import {CoreRef} from "../refs/CoreRef.sol";
+import {ICore} from "../core/ICore.sol";
+
+/// @notice Contract used by an Admin pod to manage child pods.
+
+/// @dev This contract is primarily a factory contract which an admin
+/// can use to deploy more optimistic governance pods. It will create an
+/// Orca pod and deploy an optimistic timelock alongside it.
+///
+/// The timelock and Orca pod are then linked up so that the Orca pod is
+/// the only proposer and executor.
+contract PodFactory is CoreRef, IPodFactory {
+    /// @notice Tribe roles governing deployment of pods
+    bytes32 internal constant GOVERN_ROLE = keccak256("GOVERN_ROLE");
+    bytes32 internal constant POD_DEPLOYER_ROLE =
+        keccak256("POD_DEPLOYER_ROLE");
```

Comment:
These two roles should be imported from TribeRoles instead of being redefined.

Reply:
Yeah great point